encoding/kzgrs/pointsIO.go

```diff
@@ -36,6 +37,65 @@ func ReadDesiredBytes(reader *bufio.Reader, numBytesToRead uint64) ([]byte, erro
 	return buf, nil
 }
 
+// Read the n-th G1 point from SRS.
```

Comment:
nit: in general, comments should start with the name of the method `ReadG1Point reads the n-th...`

Reply:
I'm concerned with getting everything wired up properly for now. That last PR will focus on code cleanup.